app/bundles/LeadBundle/Form/Type/ContactFrequencyType.php

```diff
@@ -40,167 +42,32 @@ public function __construct(CoreParametersHelper $coreParametersHelper)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $showContactFrequency         = $this->coreParametersHelper->getParameter('show_contact_frequency');
-        $showContactPauseDates        = $this->coreParametersHelper->getParameter('show_contact_pause_dates');
-        $showContactPreferredChannels = $this->coreParametersHelper->getParameter('show_contact_preferred_channels');
-        $showContactCategories        = $this->coreParametersHelper->getParameter('show_contact_categories');
-        $showContactSegments          = $this->coreParametersHelper->getParameter('show_contact_segments');
-
-        if (isset($options['channels']) && $options['channels']) {
+        $showContactCategories = $this->coreParametersHelper->getParameter('show_contact_categories');
+        $showContactSegments   = $this->coreParametersHelper->getParameter('show_contact_segments');
+        // var_dump($options['data'], $options['channels']);die;
+        if (!empty($options['channels'])) {
             $builder->add(
-                'subscribed_channels',
-                'choice',
+                'lead_channels',
+                'lead_contact_channels',
```

Comment:
`LeadListType::class`

Reply:
Will update.

Reply:
Should be `ContactChannelsType::class`, updated so.